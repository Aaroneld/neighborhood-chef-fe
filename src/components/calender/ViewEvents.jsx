import React from 'react';

//component imports
import CalendarView from './calender-view/CalendarView';
import EventDetails from '../shared/event-details/EventDetails';

const ViewEvents = () => {
    return (
        <div className="component-main">
            <div className="middle-calendar">
                <CalendarView />
            </div>
            <div className="right-side-calendar">
                <EventDetails />
            </div>
        </div>
    );
};

export default ViewEvents;
