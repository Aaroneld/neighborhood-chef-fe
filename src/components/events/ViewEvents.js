import React from 'react';

const ViewEvents = () => {
  return (
    <h1>ViewEvents Component</h1>
  )
}

export default ViewEvents;
