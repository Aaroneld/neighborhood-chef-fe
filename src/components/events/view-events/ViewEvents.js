import React from "react";

//component imports
import Sidebar from "../../dashboard/Sidebar";
import MonthPicker from "./MonthPicker";
import CalendarView from "./CalendarView";
import EventDetails from "./EventDetails";
import Header from "../../dashboard/Header";

const ViewEvents = () => {
  return (
    <div className="component-main">
      <Sidebar active="view-events" />
      <div className="middle-calendar">
        <MonthPicker />
        <CalendarView />
      </div>
      <div className="right-side-calendar">
        <Header />
        <EventDetails />
      </div>
    </div>
  );
};

export default ViewEvents;
