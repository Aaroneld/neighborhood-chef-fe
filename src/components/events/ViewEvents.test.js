import React from "react";
import ViewEvents from "./ViewEvents.js";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

describe("Test view event static properties", () => {
  let ViewEventsComponent;
  beforeEach(() => {
    ViewEventsComponent = render(<ViewEvents />);
  });

  test("ViewEvents component renders", () => {
    expect(ViewEventsComponent.getByText(/ViewEvents Component/i));
  });
});
