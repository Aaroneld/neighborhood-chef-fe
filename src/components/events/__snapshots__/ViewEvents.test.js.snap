// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewEvents component renders 1`] = `
<h1>
  ViewEvents Component
</h1>
`;
